src/main/java/cz/cvut/kbss/study/model/Role.java

```diff
@@ -0,0 +1,125 @@
+package cz.cvut.kbss.study.model;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import cz.cvut.kbss.jopa.model.annotations.Individual;
+import cz.cvut.kbss.study.security.SecurityConstants;
+
+public enum Role {
+
+    // TODO deprecated -- should be removed.
+    @Individual(iri=Vocabulary.s_i_RM_ADMIN)
+    administrator(SecurityConstants.administrator, Vocabulary.s_i_RM_ADMIN),
+    // TODO deprecated -- should be removed.
+    @Individual(iri = Vocabulary.s_i_RM_USER)
+    user(SecurityConstants.user, Vocabulary.s_i_RM_USER),
+
+    @Individual(iri = Vocabulary.s_i_impersonate_role)
+    impersonate(SecurityConstants.impersonate, Vocabulary.s_i_impersonate_role),
+
+    @Individual(iri = Vocabulary.s_i_delete_all_records_role)
+    deleteAllRecords(SecurityConstants.deleteAllRecords, Vocabulary.s_i_delete_all_records_role),
+
+    @Individual(iri = Vocabulary.s_i_view_all_records_role)
+    viewAllRecords(SecurityConstants.viewAllRecords, Vocabulary.s_i_view_all_records_role),
+
+    @Individual(iri = Vocabulary.s_i_edit_all_records_role)
+    editAllRecords(SecurityConstants.editAllRecords, Vocabulary.s_i_edit_all_records_role),
+
+    @Individual(iri = Vocabulary.s_i_delete_organization_records_role)
+    deleteOrganizationRecords(SecurityConstants.deleteOrganizationRecords, Vocabulary.s_i_delete_organization_records_role),
+
+    @Individual(iri = Vocabulary.s_i_view_organization_records_role)
+    viewOrganizationRecords(SecurityConstants.viewOrganizationRecords, Vocabulary.s_i_view_organization_records_role),
+
+    @Individual(iri = Vocabulary.s_i_edit_organization_records_role)
+    editOrganizationRecords(SecurityConstants.editOrganizationRecords, Vocabulary.s_i_edit_organization_records_role),
+
+    @Individual(iri = Vocabulary.s_i_edit_users_role)
+    editUsers(SecurityConstants.editUsers, Vocabulary.s_i_edit_users_role),
+
+    @Individual(iri = Vocabulary.s_i_complete_records_role)
+    completeRecords(SecurityConstants.completeRecords, Vocabulary.s_i_complete_records_role),
+
+    @Individual(iri = Vocabulary.s_i_reject_records_role)
+    rejectRecords(SecurityConstants.rejectRecords, Vocabulary.s_i_reject_records_role),
+
+    @Individual(iri = Vocabulary.s_i_publish_records_role)
+    publishRecords(SecurityConstants.publishRecords ,Vocabulary.s_i_publish_records_role),
+
+    @Individual(iri = Vocabulary.s_i_import_codelists_role)
+    importCodelists(SecurityConstants.importCodelists, Vocabulary.s_i_import_codelists_role);
+
+    private final String iri;
+
+    public final String roleName;
+
+    Role(String roleName, String iri) {
+        this.iri = iri;
+        this.roleName = roleName;
+    }
+
+    @JsonValue
+    public String getRoleName(){
+        return roleName;
+    }
+
+    public String getIri() {
+        return iri;
+    }
+
+
+    @Override
+    public String toString() {
```

Comment:
I don't like this very much. Why not use getRoleName() instead of toString()? Why do we need this toString?

The reason i don't like it is because toString() is typically used in debugger on in LOG and it outputs some info about class as well. 

See multiple classes within model package, e.g. Institution:
```
    @Override
    public String toString() {
        return "Institution{<" + uri +
                ">, name='" + name + '\'' +
                "}";
    }
```

Reply:
I didn't know the best practice for using toString(). Okay, I understand now. I replaced it with getRoleName() in other places